src/integrationtest/resources/responses/rulesTemplate.json

```diff
@@ -0,0 +1,51 @@
+[
+    {
```

Comment:
Couldn't the template files be reused from the main directory? Do we have to duplicate a lot of resource files for all the tests?